File: algorithm/14. Longest Common Prefix/14.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        if(strs.size()==0)
            return "";
        string result=strs[0];
        for(int i =1; i<strs.size();i++)
        {
            int j;
            for (j = 0; j < result.size() && j < strs[i].size(); ++ j)
            {
                if(result[j]!=strs[i][j])
                    break;
            }
            result = result.substr(0,j);
        }
        return result;
    }
};

File: algorithm/14. Longest Common Prefix/14.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        if len(strs)==0:
            return ""
        result = strs[0]
        for i in xrange(len(strs)):
            j = 0
            while j < len(result) and j < len(strs[i]):
                if result[j]!=strs[i][j]:
                    break
                j+=1
            result = result[0:j]
        return result
